exporter/postgres/queries_backrest.yml

```diff
@@ -4,24 +4,38 @@
 #
 ###
 
-ccp_backrest_last_runtime:
+ccp_backrest_last_info:
```

Comment:
This will result in dead metric. Can we keep original name or cleanup ?

Reply:
But it's no longer just returning runtime information, so the name is inaccurate. And it may return even more information that we want in the future. I made note in CHANGELOG that it will break the metric in grafana and will need to be updated.

Reply:
cleanup ?

Reply:
No need for cleanup. Especially if people do want to be able to keep the history by referring to the old metric name.